cmake/stm32/utilities.cmake

```diff
@@ -12,18 +12,20 @@ function(stm32_util_create_family_targets FAMILY)
     endif()
 
     if(NOT (TARGET STM32::${FAMILY}${CORE_C}))
+        # Remove unused code, generate thumb code and enable all warnings
+        set(STM32_COMPILE_OPTIONS
+            -mthumb -Wall -ffunction-sections -fdata-sections
+        )
+

```

Comment:
The intermediate variable has no use now that `OPTION` are gone
```suggestion

```

Reply:
Removed